lib/pos_blockchain.go

```diff
@@ -503,7 +503,7 @@ func (bc *Blockchain) processOrphanBlockPoS(block *MsgDeSoBlock) error {
 		}
 	}
 	// Okay now we have the validator set ordered by stake, we can validate the QC.
-	if err = bc.isValidPoSQuorumCertificate(block, validatorsByStake); err != nil {
+	if err = bc.isValidPoSQuorumCertificate(block, validatorsByStake, validatorsByStake); err != nil {
```

Comment:
Does the second `validatorsByStake` need to change to represent the highQC validator set?

Reply:
Will be addressed in separate PR!

Reply:
Yes! I'll follow up on it in another PR. Every time we validate an aggregate QC, we need two different validator sets:
- The validator set at the incoming block's height
- The validator set at the incoming block's parent's height